phpBB/phpbb/controller/helper.php

```diff
@@ -184,6 +184,8 @@ public function route($route, array $params = array(), $is_amp = true, $session_
 	* @param string $message The error message
 	* @param int $code The error code (e.g. 404, 500, 503, etc.)
 	* @return Response A Response instance
+	*
+	* @deprecated 3.1.3 (To be removed: 3.3.0) Use exceptions instead.
```

Comment:
We can't replace trigger_error yes, this page is only for the "Symfony's Controllers"

About the policy.... the ext team can update the phpbb.com policy if they want. And this system will be explained in the dev doc (so the phpbb.com wiki for now) when I'll find the time.